### PR TITLE
Add tutorial for default fenced frames in explainer

### PR DESCRIPTION
**Pull request by blu25**
Update the explainer to include steps on how to get Chrome working with default mode fenced frames, as well as a quick example on getting a fenced frame loaded through default mode.

### CODE REVIEW

File: explainer/use_cases.md [hunk 1]
```diff
@@ -68,6 +68,17 @@ This use case is the fenced frames with no special restrictions on the src and n
     *   Since there is no unpartitioned/cross-site data available to the fenced frame, it is not able to do any cross-site data joining.
     *   Like all use cases, the fenced frame is isolated from the embedded context via any JS window references, script access, storage access, messaging APIs etc. The fenced frame does not know the embedding site’s origin or etld+1.
 
+To begin experimenting with fenced frames in default mode, enable the following experiments in `chrome://flags`:
+* Privacy Sandbox Ads APIs
+* Enable the \`FencedFrameConfig\` constructor.
```
Comment (domfarolino) [inline explainer/use_cases.md:73]: Let's add explicit links to chrome://flags/#enable-fenced-frames-developer-mode and

File: explainer/use_cases.md [hunk 2]
```diff
@@ -68,6 +68,17 @@ This use case is the fenced frames with no special restrictions on the src and n
     *   Since there is no unpartitioned/cross-site data available to the fenced frame, it is not able to do any cross-site data joining.
     *   Like all use cases, the fenced frame is isolated from the embedded context via any JS window references, script access, storage access, messaging APIs etc. The fenced frame does not know the embedding site’s origin or etld+1.
 
+To begin experimenting with fenced frames in default mode, enable the following experiments in `chrome://flags`:
+* Privacy Sandbox Ads APIs
```
Comment (domfarolino) [inline explainer/use_cases.md:72]: When I search for exactly this text I cannot find the flag you seem to be referencing. Can you put a direct link to it?

File: explainer/use_cases.md [hunk 3]
```diff
@@ -68,6 +68,17 @@ This use case is the fenced frames with no special restrictions on the src and n
...
+* Enable the \`FencedFrameConfig\` constructor.
+
+After relaunching Chrome, a fenced frame can be added to a page with the following code:
+```
+const frame = document.createElement("fencedframe");
+const config = new FencedFrameConfig("https://yoursite.com/page.html");
```
Comment (domfarolino) [inline explainer/use_cases.md:78]: Let's use example.com which is more canonical.

File: explainer/use_cases.md [hunk 4]
```diff
@@ -68,6 +68,17 @@ This use case is the fenced frames with no special restrictions on the src and n
...
+frame.config = config;
+document.body.appendChild(frame);
```
Comment (domfarolino) [inline explainer/use_cases.md:80]: append() is suffucient

File: explainer/use_cases.md (hunk 1 above)
Comment (blu25) [reply explainer/use_cases.md:73]: I don't think you're allowed to actually have a hyperlink to internal chrome urls, so I'll just put the url in parentheses.

File: explainer/use_cases.md [hunk 5]
```diff
@@ -69,15 +69,15 @@ This use case is the fenced frames with no special restrictions on the src and n
     *   Like all use cases, the fenced frame is isolated from the embedded context via any JS window references, script access, storage access, messaging APIs etc. The fenced frame does not know the embedding site’s origin or etld+1.
 
 To begin experimenting with fenced frames in default mode, enable the following experiments in `chrome://flags`:
-* Privacy Sandbox Ads APIs
-* Enable the \`FencedFrameConfig\` constructor.
+* Privacy Sandbox Ads APIs (chrome://flags/#privacy-sandbox-ads-apis)
```
Comment (domfarolino) [inline explainer/use_cases.md:72]: I don't see this in Canary. Is it gone?

Comment (blu25) [reply explainer/use_cases.md:72]: Version 118.0.5939.0 has it for me. I also installed a fresh copy of Canary on my personal laptop and verified that the flags are there as well (same version).

Comment (domfarolino) [reply explainer/use_cases.md:72]: Oh interesting. On M117 I have to "temporarily expire M116" to see that flag. I guess it must've expired and then they quickly renewed it. Anyways, as long as it is showing up by default in >=M118 that's good.

**Review by domfarolino — APPROVED** [review]
We can close https://github.com/WICG/fenced-frame/issues/94 too right? I don't think it's mentioned in the OP

**Comment by blu25** [comment]
> We can close #94 too right? I don't think it's mentioned in the OP

Yes, I'll go ahead and close it out.

**Pull request merged**